### ISSUE TITLE
Broken after updating to Rails 7.1

### ISSUE THREAD

**Issue by hernanvicente** [OffgridElectric/rails-data-migrations#16]
I recently updated my Rails version to 7.1.1, but I have been experiencing some issues with the `create_table` method. Specifically, when running `./bin/rails data:migrate`, I am seeing the following error:

```bash
NoMethodError: undefined method `create_table' for RailsDataMigrations::LogEntry:Class

Tasks: TOP => data:migrate => data:init_migration
(See full trace by running task with --trace)
```

It seems that in Rails 7.1, `::ActiveRecord::SchemaMigration` no longer inherits from `ActionRecord::Base`. I found a solution to this problem by introducing a monkey-patch, but I would like to create a pull request with the necessary changes to fix the issue. You can find more information about the change in the SchemaMigration [here](https://github.com/rails/rails/pull/45908).

**Comment by hernanvicente** [comment]
Here is the related [PR](https://github.com/OffgridElectric/rails-data-migrations/pull/17) that solves this. I hope it helps.

**Comment by hernanvicente** [comment]
Hello, is there something I can do to have this merged?

**Comment by serggl** [comment]
fixed via #17
